create-turbo/.eslintrc.js

```diff
@@ -1,6 +1,26 @@
 module.exports = {
+  root: true,
```

Comment:
This is required so it doesn't also use the lint rules within the top level root (was seeing some nextjs lint rules coming through without this)